src/util/constants.js

```diff
@@ -19,17 +9,25 @@ export const defaultExerciseSites = [
     "subdomain": "web",
     "name": "Hellotalk"
   },
+  {
+    "hostname": "www.zeeguu.org",
+    "href": "https://www.zeeguu.org/practice/#!/practice/plan/0",
+    "pathname": "//practice/#!/practice/plan/0",
+    "regex": "*://*.www.zeeguu.org/*",
+    "tld": "org",
+    "domain": "zeeguu",
+    "subdomain": "www",
+    "name": "Zeeguu"
+  }
   // 'https://www.brainscape.com/'
 ];
 
 export const defaultExerciseSite = defaultExerciseSites[0];
```

Comment:
let's use `defaultExerciseSites[1]` -- i like zeeguu more than this hellotalk thing.

in fact, do you use it? one of the reviews of the app says: 

>   This app is under the parent company Tencent. This is why the app requests unnecessary permissions to your photos as it accesses all other data from within your storage folders WITHOUT USER CONSENT. Also why users can't create a new account on the device because it has stored IMEI illegally. Tencent...